src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt

```diff
@@ -347,4 +347,22 @@ class CommonPlatformCourtEventListenerIntTest : MessagingMultiNodeTestBase() {
       mapOf("MESSAGE_ID" to messageId, "SOURCE_SYSTEM" to COMMON_PLATFORM.name, "EVENT_TYPE" to COMMON_PLATFORM_HEARING.name),
     )
   }
+
+  @Test
+  fun `should not process when is organisation`() {
+    val youthDefendantId = randomDefendantId()
+    val messageId = publishCommonPlatformMessage(
+      commonPlatformHearing(
+        listOf(
+          CommonPlatformHearingSetup(defendantId = youthDefendantId, isYouth = null, isPerson = false),
```

Comment:
very minor point but calling the defendantId youthDefendantId and then setting isYouth to false is a bit confusing